### PR TITLE
Refactor remaining everrest websocket endpoints to use Json RPC

### PR DESCRIPTION
**Pull request by mshaposhnik**
### What does this PR do?
Refactors organization & system websocket channels to use Json RPC. Refactor CLI client in part of graceful shutdown. Remove utility classes used for everrest websockets.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/5357

#### Release Notes
Refactored organization & system websocket channels to use Json RPC

#### Docs PR
N/A

### CODE REVIEW

File: dockerfiles/lib/src/api/wsmaster/system/system.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import {Websocket} from "../../../spi/websocket/websocket";
 import {HttpJsonRequest} from "../../../spi/http/default-http-json-request";
 import {DefaultHttpJsonRequest} from "../../../spi/http/default-http-json-request";
 import {HttpJsonResponse} from "../../../spi/http/default-http-json-request";
-import {MessageBus} from "../../../spi/websocket/messagebus";
+import {JsonRpcBus} from "../../../spi/websocket/json-rpc-bus";
```
Comment (benoitf) [inline dockerfiles/lib/src/api/wsmaster/system/system.ts:18]: now, we can get rid of MessageBus class as well and all references to it.

File: dockerfiles/lib/src/api/wsmaster/system/system.ts [hunk 2]
```diff
@@ -113,6 +113,8 @@ export class System {
       });
       return messageBus.subscribeAsync(channelToListen, callbackSubscriber);
     }).then((subscribed: string) => {
+         return this.delay(2000);
```
Comment (benoitf) [inline dockerfiles/lib/src/api/wsmaster/system/system.ts]: please add a comment why it's required

File: dockerfiles/lib/src/api/wsmaster/system/system.ts (hunk 1 above)
Comment (mshaposhnik) [reply dockerfiles/lib/src/api/wsmaster/system/system.ts:18]: ok

File: dockerfiles/lib/src/api/wsmaster/system/system.ts (hunk 2 above)
Comment (mshaposhnik) [reply dockerfiles/lib/src/api/wsmaster/system/system.ts]: ok

**Review by benoitf — APPROVED** [review]

**Review by skabashnyuk — APPROVED** [review]

**Review by riuvshin — APPROVED** [review]
please test CLI carefully after these changes

File: wsagent/che-wsagent-core/src/main/java/org/eclipse/che/wsagent/server/WsAgentServletModule.java [hunk 3]
```diff
@@ -22,7 +21,5 @@
 @DynaModule
 public class WsAgentServletModule extends ServletModule {
```
Comment (garagatyi) [inline wsagent/che-wsagent-core/src/main/java/org/eclipse/che/wsagent/server/WsAgentServletModule.java:22]: Do we need this class at all?

**Review by garagatyi — APPROVED** [review]

**Pull request merged**